themes/digital.gov/src/scss/new/styles.scss

```diff
@@ -118,6 +118,7 @@
 
 // Lists of related pages
 @forward "collection";
+@forward "featured-resource";
```

Comment:
We should me this to the shortcode section since it is technically is setup to be one.